tests/code/test_events.py

```diff
@@ -523,3 +520,20 @@ def test_userWithNoInterestedEvent():
     user = "ayisie" #no interest selected
     events = getUpcomingEventsForUser(user)
     assert len(events) == 0
+
+@pytest.mark.integration
+def test_format24HourTime():
+
+    # tests valid "input times"
+    assert format24HourTime('08:00 AM')
+    assert format24HourTime('8:00 AM')
+    assert format24HourTime('05:00 PM')
+    assert format24HourTime('07:30 PM')
+    assert format24HourTime('12:00 PM')
```

Comment:
You should be checking that the function actually does this properly. right now you are just making sure there is no exception thrown.

Reply:
We corrected and expanded on the tests.